apps/portal/extensions/locales/locale_si.json

```diff
@@ -139,4 +139,6 @@
   "designer.componentProperties.priority": "පූර්වතාව",
   "bad.request" : "වැරදි ඉල්ලීමකි",
   "forbidden" : "Forbiden"
```

Comment:
This is corrected in the above file but not here. Shouldn't this be in Sinhalese and not in English?